packages/uniforms-material/src/RadioField.tsx

```diff
@@ -46,7 +46,7 @@ function Radio({
       name={name}
       onChange={(event: any) => disabled || onChange(event.target.value)}
       ref={inputRef}
-      value={value}
+      value={value || ''}
```

Comment:
I'm not sure if `RadioField` works with numbers, but `??` may be better here (consider `0`).